packages/typespec-go/src/tcgcadapter/adapter.ts

```diff
@@ -46,6 +32,20 @@ export async function tcgcToGoCodeModel(context: EmitContext<GoEmitterOptions>):
     codeModelType = 'azure-arm';
   }
 
+  const info = new go.Info('TODO Title');
+  const options = new go.Options(sdkContext.sdkPackage.licenseInfo ? sdkContext.sdkPackage.licenseInfo.header : headerText,
```

Comment:
It seems we use the TCGC info of license header. Why don't we also use the license file content?